README.md

```diff
@@ -132,6 +133,11 @@ http://localhost:2322/api?q=berlin&limit=2
 http://localhost:2322/api?q=berlin&lang=it
 ```
 
+#### Filter results by bounding box
+```
+http://localhost:2322/api?q=berlin&bbox=9.5,51.5,11.5,53.5
```

Comment:
It would be important to mention which order the bbox has. Maybe we should also use a different order with left,bottom and right,top points ala `latSouth,lonWest,latNorth,lonEast` or why did you pick this order?

Reply:
You are right, this should be documented. Concerning the order: to me this was the one I‘m used to, but [Nomination and Overpass](https://wiki.openstreetmap.org/wiki/Bounding_Box) both use a different order.
Should I change it?
And before merging, I should add a verification of the lat/lon limits.

Reply:
> Should I change it?

lon,lat,... is not a wrong choice (that's the order geojson uses I think) but the point parameter is lat,lon order and so it might be a bit inconsistent, but I'm not sure and probably not super important.

Reply:
> Should I change it?

Up, not necessary. I confused the APIs ;)

Reply:
Did you change the order now? Because I read from the code below that you expect <minLon, maxLon, minLat, maxLat>. My preference would be as documented: <minLon,minLat,maxLon,maxLat>. (Or to be precise: <x1,y1,x2,y2>, Envelope() does not seem to care if it is min,max or max,min.)

Just to mention it, the odd boundingbox format of Nominatim is only used for output. The viewbox input parameter also expects <minLon,minLat,maxLon,maxLat>.

Reply:
As you have a preference then we should do x1,y1,x2,y2